ddp-workspace/projects/ddp-pancan/src/app/components/welcome/stay-informed-section/stay-informed-section.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="container">
+    <h2 class="title static-page-title">{{(title || 'App.HomePage.StayInformedSection.Title') | translate}}</h2>
+    <div class="social-media">
+        <div class="text static-page-paragraph-text" translate>App.HomePage.StayInformedSection.Text</div>
```

Comment:
why we did title and btn text configurable but not the text?
Also do we really need to make it configurable? the text looks exactly the same. Can we just create the separate section in json for StayInformed? like App.StayInformed.Text

Reply:
Agree, technically all fields should be configurable there. 
We just started from welcome page. Then (suddenly) a new Colorectal page was added.
`text` are the same for the component, but `title` and `button text` distinguish a bit in Spanish translations 
(`welcome page` vs `colorectal page`).
Anyway, fixed.